[Bug] [S3 File] After updating to the current dev branch cluster, s3File failed to run the task

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened

After updating to the current dev branch cluster, s3File failed to run the task

### SeaTunnel Version

dev

### SeaTunnel Config

```conf
{
    "env": {
        "job.name": "Excel输入测试",
        "job.mode": "batch"
    },
    "preHandler": [

    ],
    "source": [
        {
            "plugin_name": "S3File",
            "resourceType": "file",
            "path": "/file-node/upload/excel/669634f980e02dd229a3eb04.xlsx",
            "fs.s3a.endpoint": "******",
            "fs.s3a.aws.credentials.provider": "org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider",
            "access_key": "P7HpCVa0Eth8X3dM",
            "secret_key": "xxM9cJL5QKqq9M7jvwkQ5LJgsQSVvIay",
            "bucket": "s3a://ottomi",
            "file_format_type": "excel",
            "skip_header_row_number": 1,
            "date_format": "yyyy-MM-dd",
            "datetime_format": "yyyy-MM-dd HH:mm:ss",
            "time_format": "HH:mm:ss",
            "schema": {
                "fields": {

                }
            },
            "resources": [
                "testOut_0.xlsx"
            ],
            "result_table_name": "ot_768f13511d68480b8ee8b4112f6efa6b",
            "ottomi_node_type": "ExcelSource"
        }
    ],
    "transform": [

    ],
    "sink": [
        {
            "file_format_type": "excel",
            "sheet_name": "ottomi",
            "custom_filename": true,
            "file_name_expression": "testaaa",
            "is_enable_transaction": false,
            "batch_size": 1000000000,
            "max_rows_in_memory ": 2048,
            "plugin_name": "S3File",
            "bucket": "s3a://ottomi",
            "fs.s3a.endpoint": "******",
            "fs.s3a.aws.credentials.provider": "org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider",
            "access_key": "P7HpCVa0Eth8X3dM",
            "secret_key": "xxM9cJL5QKqq9M7jvwkQ5LJgsQSVvIay",
            "tmp_path": "/tmp/ottomi",
            "path": "/file-node/download/1789908261255188482/excel/",
            "result_table_name": "ot_1e0968446fb242608701085865694ca2",
            "ottomi_node_type": "ExcelSink",
            "source_table_name": [
                "ot_768f13511d68480b8ee8b4112f6efa6b"
            ]
        }
    ]
}
```


### Running Command

```shell
bin/seatunnel.sh -c demo.json
```


### Error Exception

```log
Caused by: java.io.IOException: org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider instantiation exception: java.lang.NoSuchMethodError: com.google.common.base.Preconditions.checkArgument(ZLjava/lang/String;Ljava/lang/Object;Ljava/lang/Object;)V
        at org.apache.hadoop.fs.s3a.S3AUtils.createAWSCredentialProvider(S3AUtils.java:678) ~[hadoop-aws-3.1.4.jar:?]
        at org.apache.hadoop.fs.s3a.S3AUtils.createAWSCredentialProviderSet(S3AUtils.java:566) ~[hadoop-aws-3.1.4.jar:?]
        at org.apache.hadoop.fs.s3a.DefaultS3ClientFactory.createS3Client(DefaultS3ClientFactory.java:52) ~[hadoop-aws-3.1.4.jar:?]
        at org.apache.hadoop.fs.s3a.S3AFileSystem.initialize(S3AFileSystem.java:256) ~[hadoop-aws-3.1.4.jar:?]
        at org.apache.hadoop.fs.FileSystem.createFileSystem(FileSystem.java:3303) ~[seatunnel-hadoop3-3.1.4-uber.jar:2.3.6-SNAPSHOT]
        at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:476) ~[seatunnel-hadoop3-3.1.4-uber.jar:2.3.6-SNAPSHOT]
        at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:227) ~[seatunnel-hadoop3-3.1.4-uber.jar:2.3.6-SNAPSHOT]
        at org.apache.seatunnel.connectors.seatunnel.file.hadoop.HadoopFileSystemProxy.initialize(HadoopFileSystemProxy.java:222) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.hadoop.HadoopFileSystemProxy.getFileSystem(HadoopFileSystemProxy.java:171) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.hadoop.HadoopFileSystemProxy.listStatus(HadoopFileSystemProxy.java:154) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.source.reader.AbstractReadStrategy.getFileNamesByPath(AbstractReadStrategy.java:94) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.config.BaseFileSourceConfig.parseFilePaths(BaseFileSourceConfig.java:73) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.config.BaseFileSourceConfig.<init>(BaseFileSourceConfig.java:64) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.s3.source.config.S3FileSourceConfig.<init>(S3FileSourceConfig.java:44) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.s3.source.config.MultipleTableS3FileSourceConfig.getBaseSourceConfig(MultipleTableS3FileSourceConfig.java:32) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.config.BaseMultipleTableFileSourceConfig.parseFromFileSourceConfig(BaseMultipleTableFileSourceConfig.java:54) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.config.BaseMultipleTableFileSourceConfig.<init>(BaseMultipleTableFileSourceConfig.java:39) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.s3.source.config.MultipleTableS3FileSourceConfig.<init>(MultipleTableS3FileSourceConfig.java:27) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.s3.source.S3FileSource.<init>(S3FileSource.java:28) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.s3.source.S3FileSourceFactory.lambda$createSource$0(S3FileSourceFactory.java:48) ~[?:?]
        at org.apache.seatunnel.api.table.factory.FactoryUtil.createAndPrepareSource(FactoryUtil.java:112) ~[seatunnel-starter.jar:2.3.6-SNAPSHOT]
        at org.apache.seatunnel.api.table.factory.FactoryUtil.createAndPrepareSource(FactoryUtil.java:73) ~[seatunnel-starter.jar:2.3.6-SNAPSHOT]
        ... 14 more
Caused by: java.lang.NoSuchMethodError: com.google.common.base.Preconditions.checkArgument(ZLjava/lang/String;Ljava/lang/Object;Ljava/lang/Object;)V
        at org.apache.hadoop.fs.s3a.S3AUtils.lookupPassword(S3AUtils.java:742) ~[hadoop-aws-3.1.4.jar:?]
        at org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider.<init>(SimpleAWSCredentialsProvider.java:59) ~[hadoop-aws-3.1.4.jar:?]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_212]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_212]
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_212]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_212]
        at org.apache.hadoop.fs.s3a.S3AUtils.createAWSCredentialProvider(S3AUtils.java:635) ~[hadoop-aws-3.1.4.jar:?]
        at org.apache.hadoop.fs.s3a.S3AUtils.createAWSCredentialProviderSet(S3AUtils.java:566) ~[hadoop-aws-3.1.4.jar:?]
        at org.apache.hadoop.fs.s3a.DefaultS3ClientFactory.createS3Client(DefaultS3ClientFactory.java:52) ~[hadoop-aws-3.1.4.jar:?]
        at org.apache.hadoop.fs.s3a.S3AFileSystem.initialize(S3AFileSystem.java:256) ~[hadoop-aws-3.1.4.jar:?]
        at org.apache.hadoop.fs.FileSystem.createFileSystem(FileSystem.java:3303) ~[seatunnel-hadoop3-3.1.4-uber.jar:2.3.6-SNAPSHOT]
        at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:476) ~[seatunnel-hadoop3-3.1.4-uber.jar:2.3.6-SNAPSHOT]
        at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:227) ~[seatunnel-hadoop3-3.1.4-uber.jar:2.3.6-SNAPSHOT]
        at org.apache.seatunnel.connectors.seatunnel.file.hadoop.HadoopFileSystemProxy.initialize(HadoopFileSystemProxy.java:222) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.hadoop.HadoopFileSystemProxy.getFileSystem(HadoopFileSystemProxy.java:171) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.hadoop.HadoopFileSystemProxy.listStatus(HadoopFileSystemProxy.java:154) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.source.reader.AbstractReadStrategy.getFileNamesByPath(AbstractReadStrategy.java:94) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.config.BaseFileSourceConfig.parseFilePaths(BaseFileSourceConfig.java:73) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.config.BaseFileSourceConfig.<init>(BaseFileSourceConfig.java:64) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.s3.source.config.S3FileSourceConfig.<init>(S3FileSourceConfig.java:44) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.s3.source.config.MultipleTableS3FileSourceConfig.getBaseSourceConfig(MultipleTableS3FileSourceConfig.java:32) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.config.BaseMultipleTableFileSourceConfig.parseFromFileSourceConfig(BaseMultipleTableFileSourceConfig.java:54) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.config.BaseMultipleTableFileSourceConfig.<init>(BaseMultipleTableFileSourceConfig.java:39) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.s3.source.config.MultipleTableS3FileSourceConfig.<init>(MultipleTableS3FileSourceConfig.java:27) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.s3.source.S3FileSource.<init>(S3FileSource.java:28) ~[?:?]
        at org.apache.seatunnel.connectors.seatunnel.file.s3.source.S3FileSourceFactory.lambda$createSource$0(S3FileSourceFactory.java:48) ~[?:?]
        at org.apache.seatunnel.api.table.factory.FactoryUtil.createAndPrepareSource(FactoryUtil.java:112) ~[seatunnel-starter.jar:2.3.6-SNAPSHOT]
        at org.apache.seatunnel.api.table.factory.FactoryUtil.createAndPrepareSource(FactoryUtil.java:73) ~[seatunnel-starter.jar:2.3.6-SNAPSHOT]
        ... 14 more
```


### Zeta or Flink or Spark Version

2.3.6-dev

### Java or Scala Version

1.8

### Screenshots

![image](https://github.com/user-attachments/assets/38a9a189-1805-430e-8c15-91322c107f7a)
![image](https://github.com/user-attachments/assets/4514628d-8f6a-49ed-b63f-3a4e0e5c224f)


### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/pom.xml
+++ b/pom.xml
@@ -158,6 +158,8 @@
         <prometheus.simpleclient.version>0.16.0</prometheus.simpleclient.version>
         <enableSourceJarCreation>true</enableSourceJarCreation>
 
+        <hadoop-aws.version>3.1.4</hadoop-aws.version>
+
     </properties>
 
     <dependencyManagement>

--- a/seatunnel-connectors-v2/connector-file/connector-file-s3/pom.xml
+++ b/seatunnel-connectors-v2/connector-file/connector-file-s3/pom.xml
@@ -29,21 +29,6 @@
     <artifactId>connector-file-s3</artifactId>
     <name>SeaTunnel : Connectors V2 : File : S3</name>
 
-    <properties>
-        <hadoop-aws.version>3.1.4</hadoop-aws.version>
-        <guava.version>27.0-jre</guava.version>
-    </properties>
-
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>com.google.guava</groupId>
-                <artifactId>guava</artifactId>
-                <version>${guava.version}</version>
-            </dependency>
-        </dependencies>
-    </dependencyManagement>
-
     <dependencies>
 
         <dependency>
@@ -67,14 +52,10 @@
         </dependency>
 
         <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-        </dependency>
-
-        <dependency>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-aws</artifactId>
-            <version>${hadoop-aws.version}</version>
+            <groupId>org.apache.seatunnel</groupId>
+            <artifactId>seatunnel-hadoop-aws</artifactId>
+            <version>${project.version}</version>
+            <classifier>optional</classifier>
             <scope>provided</scope>
             <exclusions>
                 <exclusion>

--- a/seatunnel-dist/pom.xml
+++ b/seatunnel-dist/pom.xml
@@ -104,7 +104,6 @@
                 <!-- Imap storage dependency package  -->
                 <hadoop-aliyun.version>3.1.4</hadoop-aliyun.version>
                 <json-smart.version>2.4.7</json-smart.version>
-                <hadoop-aws.version>3.1.4</hadoop-aws.version>
                 <aws-java-sdk.version>1.11.271</aws-java-sdk.version>
                 <netty-buffer.version>4.1.89.Final</netty-buffer.version>
                 <hive.exec.version>3.1.3</hive.exec.version>
@@ -735,9 +734,10 @@
 
                 <!-- hadoop jar -->
                 <dependency>
-                    <groupId>org.apache.hadoop</groupId>
-                    <artifactId>hadoop-aws</artifactId>
-                    <version>${hadoop-aws.version}</version>
+                    <groupId>org.apache.seatunnel</groupId>
+                    <artifactId>seatunnel-hadoop-aws</artifactId>
+                    <version>${project.version}</version>
+                    <classifier>optional</classifier>
                     <scope>provided</scope>
                 </dependency>
                 <dependency>
@@ -958,6 +958,14 @@
                     <groupId>org.apache.seatunnel</groupId>
                     <artifactId>connector-sls</artifactId>
                     <version>${project.version}</version>
+                    <scope>provided</scope>
+                </dependency>
+
+                <dependency>
+                    <groupId>org.apache.seatunnel</groupId>
+                    <artifactId>seatunnel-hadoop-aws</artifactId>
+                    <version>${project.version}</version>
+                    <classifier>optional</classifier>
                     <scope>provided</scope>
                 </dependency>
             </dependencies>

--- a/seatunnel-dist/src/main/assembly/assembly-bin-ci.xml
+++ b/seatunnel-dist/src/main/assembly/assembly-bin-ci.xml
@@ -161,7 +161,7 @@
             <scope>provided</scope>
         </dependencySet>
 
-        <!-- =================== JDBC Connector Drivers  And SeaTunnel Hadoop3 Uber Jar ===================  -->
+        <!-- =================== JDBC Connector Drivers, SeaTunnel Hadoop3 Uber Jar And SeaTunnel Hadoop AWS Uber Jar ===================  -->
         <dependencySet>
             <useProjectArtifact>false</useProjectArtifact>
             <useTransitiveDependencies>true</useTransitiveDependencies>
@@ -181,9 +181,9 @@
                 <include>com.amazon.redshift:redshift-jdbc42:jar</include>
                 <include>net.snowflake.snowflake-jdbc:jar</include>
                 <include>com.xugudb:xugu-jdbc:jar</include>
-                <include>org.apache.hadoop:hadoop-aws:jar</include>
                 <include>com.amazonaws:aws-java-sdk-bundle:jar</include>
                 <include>org.apache.seatunnel:seatunnel-hadoop3-3.1.4-uber:jar:*:optional</include>
+                <include>org.apache.seatunnel:seatunnel-hadoop-aws:jar:*:optional</include>
                 <!--Add hadoop aliyun jar -->
                 <include>org.apache.hadoop:hadoop-aliyun:jar</include>
                 <include>com.aliyun.oss:aliyun-sdk-oss:jar</include>

--- a/seatunnel-dist/src/main/assembly/assembly-bin.xml
+++ b/seatunnel-dist/src/main/assembly/assembly-bin.xml
@@ -174,6 +174,19 @@
             <scope>provided</scope>
         </dependencySet>
 
+        <!-- ============ SeaTunnel Hadoop-AWS Uber Jar============  -->
+        <dependencySet>
+            <useProjectArtifact>false</useProjectArtifact>
+            <useTransitiveDependencies>true</useTransitiveDependencies>
+            <unpack>false</unpack>
+            <includes>
+                <include>org.apache.seatunnel:seatunnel-hadoop-aws:jar:*:optional</include>
+            </includes>
+            <outputFileNameMapping>${artifact.file.name}</outputFileNameMapping>
+            <outputDirectory>/lib</outputDirectory>
+            <scope>provided</scope>
+        </dependencySet>
+
         <!-- ============ Connectors Jars And Transforms V2 Jar ============  -->
         <!-- SeaTunnel connectors for Demo -->
         <dependencySet>

--- a/seatunnel-shade/pom.xml
+++ b/seatunnel-shade/pom.xml
@@ -35,6 +35,7 @@
         <module>seatunnel-hazelcast</module>
         <module>seatunnel-janino</module>
         <module>seatunnel-jetty9-9.4.56</module>
+        <module>seatunnel-hadoop-aws</module>
 
     </modules>
 

--- a/seatunnel-shade/seatunnel-hadoop-aws/pom.xml
+++ b/seatunnel-shade/seatunnel-hadoop-aws/pom.xml
@@ -1,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.seatunnel</groupId>
+        <artifactId>seatunnel-shade</artifactId>
+        <version>${revision}</version>
+    </parent>
+
+    <artifactId>seatunnel-hadoop-aws</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-aws</artifactId>
+            <version>${hadoop-aws.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <finalName>seatunnel-hadoop-aws</finalName>
+                            <createSourcesJar>${enableSourceJarCreation}</createSourcesJar>
+                            <shadeSourcesContent>true</shadeSourcesContent>
+                            <shadedArtifactAttached>false</shadedArtifactAttached>
+                            <createDependencyReducedPom>false</createDependencyReducedPom>
+                            <filters>
+                                <filter>
+                                    <artifact>*:*</artifact>
+                                    <excludes>
+                                        <exclude>META-INF/*.SF</exclude>
+                                        <exclude>META-INF/*.DSA</exclude>
+                                        <exclude>META-INF/*.RSA</exclude>
+                                    </excludes>
+                                </filter>
+                            </filters>
+                            <relocations>
+                                <relocation>
+                                    <pattern>com.google.common</pattern>
+                                    <shadedPattern>${seatunnel.shade.package}.com.google.common</shadedPattern>
+                                    <includes>
+                                        <include>com.google.common.base.*</include>
+                                        <include>com.google.common.cache.*</include>
+                                        <include>com.google.common.collect.*</include>
+                                    </includes>
+                                </relocation>
+                            </relocations>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>attach-artifacts</id>
+                        <goals>
+                            <goal>attach-artifact</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <artifacts>
+                                <artifact>
+                                    <file>${basedir}/target/seatunnel-hadoop-aws.jar</file>
+                                    <type>jar</type>
+                                    <classifier>optional</classifier>
+                                </artifact>
+                            </artifacts>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>

